README.md

```diff
@@ -6,6 +6,44 @@ We are building an ecosystem of preprepocessing pipeline tools for Data Scientis
 and Data Engineers so they may quickly work through the challenge of extracting
 structured data from unstructured raw documents.
 
+## Getting Started
+
+Unstructured's open-source packages currently target Python 3.8. If you are using or contributing
+to Unstructured code, we encourage you to work with Python 3.8 in a virtual environment. You can
+use the following instructions to get up and running with a Python 3.8 virtual environemtn
+with `pyenv-virtualenv`:
+
+#### Mac / Homebrew
+
+1. Install `pyenv-virtualenv` with `brew install pyenv-virtualenv`
```

Comment:
I'm not clear because I don't remember what process I followed, do you not need to `brew install pyenv` first?

Reply:
Just added that in. Don't quite remember if you need it, but I don't think it hurts either way.